README.md

```diff
@@ -1,15 +1,24 @@
-# .NET Runtime
+# .NET Runtime <!-- omit in toc -->
```

Comment:
Why do we need these "<!-- omit in toc -->" tags? They look ugly if you are looking at raw .md.

Reply:
Those are going to be removed before merging. They were just for a table of contents extension for Markdown I used.